mesh_map/src/mesh_map.cpp

```diff
@@ -307,6 +358,7 @@ bool MeshMap::readMap()
     return false;
   }
 
+  // why?
   sleep(1);
```

Comment:
Maybe it had something to do with the service code that you removed.
I think we should remove the sleep. It looks very suspicious.

Reply:
It's removed now